bitnami/magento/values.yaml

```diff
@@ -131,212 +156,168 @@ extraEnvVarsCM:
-        value: "{{ include \"magento.host\" . }}"
-  initialDelaySeconds: 30
-  periodSeconds: 5
-  timeoutSeconds: 3
-  successThreshold: 1
-  failureThreshold: 3
+sidecars: []
 
-## Custom Liveness probes for Magento
+## Tolerations for pod assignment. Evaluated as a template.
+## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
 ##
-customLivenessProbe: {}
+tolerations: []
 
-## Custom Rediness probes Magento
+## Use existing secret for the application password
 ##
-customReadinessProbe: {}
+existingSecret:
 
-## Pod extra labels
-## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
 ##
-podLabels: {}
-
-## Annotations for server pods.
-## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+## External database configuration
 ##
-podAnnotations: {}
+externalDatabase:
+  ## Database host
+  ##
+  host:
 
-## Pod affinity preset
-## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity
-## Allowed values: soft, hard
-##
-podAffinityPreset: ""
+  ## Database host
+  ##
+  port: 3306
 
-## Pod anti-affinity preset
-## Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity
-## Allowed values: soft, hard
-##
-podAntiAffinityPreset: soft
+  ## Database user
+  ##
+  user: bn_magento
 
-## Node affinity preset
-## Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity
-## Allowed values: soft, hard
-##
-nodeAffinityPreset:
-  ## Node affinity type
-  ## Allowed values: soft, hard
-  type: ""
-  ## Node label key to match
-  ## E.g.
-  ## key: "kubernetes.io/e2e-az-name"
+  ## Database password
   ##
-  key: ""
-  ## Node label values to match
-  ## E.g.
-  ## values:
-  ##   - e2e-az1
-  ##   - e2e-az2
+  password:
+
+  ## Database name
   ##
-  values: []
+  database: bitnami_magento
 
-## Affinity for pod assignment. Evaluated as a template.
-## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
-## Note: podAffinityPreset, podAntiAffinityPreset, and nodeAffinityPreset will be ignored when it's set
+## MariaDB chart configuration
 ##
-affinity: {}
-
-## Node labels for pod assignment. Evaluated as a template.
-## ref: https://kubernetes.io/docs/user-guide/node-selection/
+## https://github.com/bitnami/charts/blob/master/bitnami/mariadb/values.yaml
 ##
-nodeSelector: {}
+mariadb:
+  ## Whether to deploy a mariadb server to satisfy the applications database requirements. To use an external database set this to false and configure the externalDatabase parameters
+  ##
+  enabled: true
+  ## Override MariaDB default image as 10.5 is not supported https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements.html#database
+  ## ref: https://github.com/bitnami/bitnami-docker-mariadb
+  image:
+    registry: docker.io
+    repository: bitnami/mariadb
```

Comment:
This is not needed since they are the default values in the subchart, only setting the tag should be enough